### DISCUSSION TITLE
Has anyone been rejected from App Store for spam?

### DISCUSSION THREAD

**Discussion by andrisole92** [infinitered/ignite#2633] [General]
We noticed your app shares a similar binary, metadata, and/or concept as apps submitted to the App Store by other developers, with only minor differences. 

This is the response I got from Apple

**Comment by andrisole92** [comment]
Submitting similar or repackaged apps is a form of spam that creates clutter and makes it difficult for users to discover new apps.

Next Steps

Since we do not accept spam apps on the App Store, we encourage you to review your app concept and submit a unique app with distinct content and functionality. 

Resources

Some factors that contribute to a spam rejection may include:

- Submitting an app with the same source code or assets as other apps already submitted to the App Store
- Creating and submitting multiple similar apps using a repackaged app template 
- Purchasing an app template with problematic code from a third party 
- Submitting several similar apps across multiple accounts

Learn more about our requirements to prevent spam in [App Store Review Guideline 4.3](https://developer.apple.com/app-store/review/guidelines/#spam).
Please see attached screenshot for details.

**Comment by markrickert** [comment]
Hey @andrisole92 sorry to hear that you're having trouble with Apple reviews. We haven't ever heard of an app being rejected for this reason *only* because it was using the ignite boilerplate.

Without knowing what type of application you submitted or how different it is from the stock boilerplate application (or other applications that use Ignite), I'm not sure there's anything we as the Ignite boilerplate development team do here.

Since we have no insight into how Apple reviews apps or the tools it uses to analyze uploaded app binaries, I'm afraid there's not a lot we can do here other than recommend that you maybe ensure that any boilerplate images and files that you aren't actually using in the app are removed from the final production bundle. 

I'm going to close this for now. Feel free to reopen if you are able to determine that the Ignite boilerplate is the reason for the app's rejection.
